salt/prometheus/files/conf-prometheus.yml

```diff
@@ -16,7 +16,7 @@ scrape_configs:
     'cove-ocds-2': 'ocp18.open-contracting.org',
     'credere': 'ocp22.open-contracting.org',
     'ocds-kingfisher-replica': 'ocp05.open-contracting.org',
-    'ocds-kingfisher2': 'ocp04.open-contracting.org',
+    'ocds-kingfisher': 'ocp23.open-contracting.org',
```

Comment:
@jpmckinney I'd like to standardise Prometheus so that it matches the salt roster (`kingfisher-main`), unless there is a reason not to?

Reply:
Fine with me :)